overlord/managers_test.go

```diff
@@ -160,6 +152,41 @@ func (ms *mgrsSuite) SetUpTest(c *C) {
 	err = os.MkdirAll(filepath.Dir(snapSeccompPath), 0755)
 	c.Assert(err, IsNil)
 	ms.snapSeccomp = testutil.MockCommand(c, snapSeccompPath, "")
+
+	o, err := overlord.New()
+	c.Assert(err, IsNil)
+	ms.o = o
+	st := ms.o.State()
+	st.Lock()
+	defer st.Unlock()
+	st.Set("seeded", true)
+
+	// add "core" snap declaration
+	headers := map[string]interface{}{
+		"series":       "16",
+		"snap-name":    "core",
+		"publisher-id": "can0nical",
+		"timestamp":    time.Now().Format(time.RFC3339),
+	}
+	headers["snap-id"] = fakeSnapID(headers["snap-name"].(string))
+	a, err := ms.storeSigning.RootSigning.Sign(asserts.SnapDeclarationType, headers, nil, "")
```

Comment:
usually that means the test is missing to add the store key in the local database, I would need to look more carefully. But SnapDeclarations are not signed by the root key so it's better to keep things realistic and use the store key.